test/system/002-help.bats

```diff
@@ -9,10 +9,30 @@ setup() {
 }
 
 @test "help: Run command 'help'" {
+  if ! hash man 2>/dev/null; then
```

Comment:
`command -v man` might be a better option than `hash man`.  While `hash` does get the job done, it doesn't seem to be the most appropriate tool for the job, and `command` is more obvious when reading.

Reply:
Ah, right. Forgot about the existence of `command`.

Reply:
Submitted https://github.com/containers/toolbox/pull/922